lib/translators/ui5Framework.js

```diff
@@ -71,7 +71,7 @@ const utils = {
 			// Ignoring UI5 Framework libraries in dependencies
 			return ui5Dependencies;
 		}
-		if (project.framework && project.framework.libraries) {
+		if (project.specVersion === "2.0" && project.framework && project.framework.libraries) {
```

Comment:
The message logged in the else block of this if is not completely correct anymore. Maybe we can log something different if the project defines an incorrect specVersion?